src/devices/Max31865/PlatinumResistanceThermometerType.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Iot.Device.Max31865
+{
+    /// <summary>
+    /// Type of Platinum Thermometer temperature sensor
+    /// </summary>
+    public enum PlatinumResistanceThermometerType : short
+    {
+        /// <summary>
+        /// Platinum Thermometer 100 temperature sensor
+        /// </summary>
+        PT100 = 100,
```

Comment:
The naming convention will go for `Pt100` rather than `PT100`. Maybe a name like `Type100` and `Type1000` would work just fine?

Reply:
I would go for Pt100 rather than type100 as the sensors are known as PT100